pipelines/ApPipe.yaml

```diff
@@ -14,8 +13,17 @@ tasks:
             detection.thresholdValue: 5.0  # needed with doDecorrelation
             # Don't have source catalogs for templates
             doSelectSources: False
+            # Scale diffim variance instead of template variance
+            doScaleDiffimVariance: True
     diaPipe:
         class: lsst.ap.association.DiaPipelineTask
+        config:
+          # Remember to run make_apdb.py with the same isolation_level and db_url as here
+          apdb.isolation_level: 'READ_UNCOMMITTED'
```

Comment:
Interesting - I added it here because `ap_association` fails if you don't specify `READ_UNCOMMITTED` (I _think_ that's true for both DB types), and I want the pipeline to work. I routinely set this config for both sqlite and postgres.

Reply:
Ok, that's surprising -- I was definitely under the impression that this setting was only required for SQLite. It's certainly only [enforced for SQLite](https://github.com/lsst/dax_apdb/blob/master/python/lsst/dax/apdb/apdb.py#L205)...

Reply:
OK, I think you're right it's only required for sqlite. I guess I can make the `isolation_level` a comment (like the `connection_timeout`) instead of an actual config, but I worry about decisions like this leading to more people messaging me when they try to run the AP Pipeline with default settings and it doesn't "just work." I wish we had a "diaPipeUsesWhateverConfigsMakeApdbUsed=True" button I could just turn on.

Reply:
Yeah, maybe we should revisit making the APDB location part of the task config. Configuring something that is *always* workspace- or workflow-dependent is not what either `CmdLineTask` or `PipelineTask` were designed for. Though I don't know what else we can do, especially in Gen 3...